cloudify_rest_client/executions.py

```diff
@@ -255,3 +255,18 @@ def cancel(self, execution_id, force=False, kill=False):
                                  data={'action': action},
                                  expected_status_code=200)
         return self._wrapper_cls(response)
+
+    def resume(self, execution_id, force=False):
+        """Resume an execution.
+
+        :param execution_id: Id of the execution to resume.
+        :param force: Whether to resume failed/cancelled executions by
+                      retrying their failed tasks.
+        :return: Resumed execution.
+        """
+        uri = '/{self._uri_prefix}/{id}'.format(self=self, id=execution_id)
+        action = 'force-resume' if force else 'resume'
+        response = self.api.post(uri,
+                                 data={'action': action},
```

Comment:
Why pass a string and not a bool?

Reply:
because executions' POST already works in terms of the 'action' command (it can take "cancel", "force-cancel", "kill", "resume", and now also "force-resume")